proof/refine/RISCV64/Untyped_R.thy

```diff
@@ -4324,9 +4324,8 @@ lemma resetUntypedCap_corres:
               apply (subst is_aligned_weaken[OF is_aligned_mult_triv2])
                apply (simp add: Kernel_Config.resetChunkBits_def minUntypedSizeBits_def)
               apply (subst unat_mult_simple)
-               apply (clarsimp)
                apply (rule order_less_trans[rotated],
-                      rule_tac n=sz in power_strict_increasing, simp+)
+                      rule_tac n=sz in power_strict_increasing, (simp add: word_bits_def)+)
```

Comment:
Does `;` work here instead of `, (..)+`? (The latter is an older idiom from before `;` -- generally if `;` works it is nicer, because it won't spill out into other subgoals)